korge/src/commonTest/kotlin/com/soywiz/korge/view/ImageAnimationViewTest.kt

```diff
@@ -13,23 +13,63 @@ import kotlin.test.assertEquals
 class ImageAnimationViewTest {
     private val animImages4Layers1 = ImageAnimation(
         frames = listOf(
```

Comment:
Is this file reformatted only? Or changes something? Because since it has a lot of things I cannot say for sure

Reply:
auto reformatted